### PR TITLE
password reveal input

### PR DESCRIPTION
**Pull request by phpsa**

### CODE REVIEW

File: content/plugins/craig-g-smith-password-reveal.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Filament Password Input
```
Comment (danharrin) [inline content/plugins/craig-g-smith-password-reveal.md:2]:
```suggestion
name: Password Input
```

File: content/plugins/craig-g-smith-password-reveal.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+slug: craig-g-smith-password-reveal
+author_slug: craig-g-smith
+categories: [form-field]
```
Comment (danharrin) [inline content/plugins/craig-g-smith-password-reveal.md:5]:
```suggestion
categories: [form-builder, form-field]
```

File: content/plugins/craig-g-smith-password-reveal.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+description: Password input to allow for masking / unmasking, auto-generate and copy
+discord_url:
```
Comment (danharrin) [inline content/plugins/craig-g-smith-password-reveal.md:7]:
```suggestion
discord_url: https://discord.com/channels/883083792112300104/970412812990033971
```

File: content/plugins/images/craig-g-smith-filament-password-reveal.png
Comment (danharrin) [inline content/plugins/images/craig-g-smith-filament-password-reveal.png:1]: I believe this may be too wide for our website and will be cropped?

**Review by danharrin — CHANGES_REQUESTED** [review]

File: content/plugins/images/craig-g-smith-filament-password-reveal.png
Comment (phpsa) [reply content/plugins/images/craig-g-smith-filament-password-reveal.png:1]: Thanks, Updated with your suggestions.

**Comment by danharrin** [comment]
Sorry, that image is still too wide. It should be 16:9 to fit on the website. ATM, it would get cropped like this, which we would not publish

Screenshot 2023-08-13 at 21 26 55

**Comment by danharrin** [comment]
Hi @phpsa did you see my comment above?

**Comment by pxlrbt** [comment]
@danharrin Btw. there is also an open PR for revealable password fields in core: https://github.com/filamentphp/filament/pull/7492#issuecomment-1683714571

**Comment by danharrin** [comment]
yeah, this plugin can still be discoverable for v2 users though. not even sure if I like that PR really

**Comment by danharrin** [comment]
No updates, please reopen this when you fix https://github.com/filamentphp/filamentphp.com/pull/111#issuecomment-1676455219

**Pull request closed without merging**